src/components/measurements/index.tsx

```diff
@@ -25,24 +25,40 @@ import {
   layout,
   jitterRawMeansByColorBy
 } from "./measurementsD3";
```

Comment:
> I'm delaying converting the measurementsD3.js file because the import of event from d3-selection causes an error. This is likely due to the d3 types not matching the version of d3-selection.¹

#1913 fixed the version mismatch, so maybe it's worth another shot. But that's a lot of functions to type, I don't think it needs to happen in this PR!

Reply:
Will revisit as part of https://github.com/nextstrain/auspice/issues/1918